src/components/Company/AssignSignatory/CreateSignatory/CreateSignatoryForm.tsx

```diff
@@ -63,22 +63,19 @@ export const CreateSignatoryForm = () => {
         </header>
 
         <Grid gridTemplateColumns={{ base: '1fr', small: ['1fr', '1fr'] }} gap={20}>
-          <TextField
-            control={control}
+          <TextInputField
             name="firstName"
             label={t('signatoryDetails.firstName')}
             isRequired
             errorMessage={t('validations.firstName')}
           />
-          <TextField
-            control={control}
+          <TextInputField
             name="lastName"
             label={t('signatoryDetails.lastName')}
             isRequired
             errorMessage={t('validations.lastName')}
           />
-          <TextField
-            control={control}
```

Comment:
We should update `disabled` passed to the `inputProps` on this one to use the `isDisabled` prop on the root component instead

Reply:
Fixed